clustpy/deep/neural_networks/_abstract_autoencoder.py

```diff
@@ -81,6 +84,18 @@ def forward(self, x: torch.Tensor) -> torch.Tensor:
         forwarded = self.block(x)
         return forwarded
 
+    def get_linear_layers(self) -> np.ndarray:
```

Comment:
instead of inferring this, you could add ids during the construction of the layers in FullyConnectedBlock?